src/components/spaces/show/header.js

```diff
@@ -65,11 +105,24 @@ export class SpaceHeader extends Component {
                 {ownerName}
               </a>
             }
-            {editableByMe &&
+
+            {isPrivate && editableByMe &&
+              <ShareableLinkOption
+                shareableLinkUrl={shareableLinkUrl}
+                onEnable={onEnableShareableLink}
+                onDisable={onDisableShareableLink}
+                onRotate={onRotateShareableLink}
+              >
+                <div className='ui image label' >
+                  <Icon name='link'/> Link
```

Comment:
I think this will work; Shareable link would be nice, but we are low on space here.